src/js/lib/project-manager/src/build.ts

```diff
@@ -0,0 +1,107 @@
+const fss = require('fs')
```

Comment:
no sections nor documentqtion in this file. Lets use the same docs rules as in Rust here (module docs + pub functions docs)